install.sh

```diff
@@ -82,3 +82,27 @@ Halide_CoreIR/test/scripts/install_travis.sh
 # if there are any missing solvers, downloads from Makai's AFS
 . ./smt-pnr/util/get_smt_solvers.sh
 pip install -e smt-pnr/package 
+
+# need this for the new dot-compare test(s)
+# pip install pygtk
+sudo apt-get install python-gtk2
+
+# hello?  is this thing on???
+
+
+# VIRT=$PWD/CGRAFlowPy3Env
+# cd CGRAFlowPy3Env
+# 
+# wget http://pypi.python.org/packages/source/P/PyGTK/pygtk-2.24.0.tar.bz2
+# tar -xf pygtk-2.24.0.tar.bz2
+# cd pygtk-2.24.0
+# 
+# # export PKG_CONFIG_PATH=/home/PATH/TO/VIRT/lib/pkgconfig
+# export PKG_CONFIG_PATH=$VIRT/lib/pkgconfig
+# 
+# # ./configure --prefix=/home/PATH/TO/VIRT/
+# ./configure --prefix=$VIRT/
+# 
+# make 
+# make install
+# 
```

Comment:
Do we need to keep this commented code around? Would be okay if you're planning to integrate it later, but it seems like this is just an alternative way to build/install pytgtk (installed using apt above).